controller/kv/helper.go

```diff
@@ -335,7 +353,7 @@ func newClusterHelper(id, version string, persist bool, keyRotationDuration time
 }
 
 func GetClusterHelper() ClusterHelper {
-	return clusHelperImpl
+	return clusHelper
```

Comment:
This should not be changed to `return clusHelper`
Member fields of var type in `clusHelper` are all empty which breaks other functions